src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -208,6 +209,29 @@ internal CookieContainer CookieContainer
             }
         }
 
+        internal int MaxAutomaticRedirections
+        {
+            get
+            {
+                return _maxAutomaticRedirections;
+            }
+
```

Comment:
In which case that should be applied consistently, e.g. MaxAutomaticRedirections below should have one, too.